CryptoCore/Controllers/HomeController.cs

```diff
@@ -365,6 +365,56 @@ public async Task<List<CoinTickerCombinedModel>> SearchBySymbol(string symbol =
             }
             return searchedCoin;
         }
+        private async Task<List<WalletDAL>> BuildWallet()
+        {
+            var userId = _userManager.GetUserId(User);
+            var WalletInfo = new List<WalletDAL>();
+
+            var listOfCoins = _db.AllWalletInfo.Where(e => e.UserID.Equals(userId)).ToList(); // TODO - Hard Coding User id
+            foreach (var followedCoin in listOfCoins)
+            {
+
+                var coinInfo = await SearchBySymbolExact(followedCoin.Symbol);
+                var tempObject = new WalletDAL();
```

Comment:
```suggestion
            var listOfCoins = _db.AllWalletInfo.Where(e => e.UserID.Equals(userId)).ToList();
            foreach (var followedCoin in listOfCoins)
            {

                var tempObject = new WalletDAL();
```

We can remove the comments since we're not hard coding userId anymore. Also what is the purpose of coinInfo? 
We're not using that anywhere but we're awaiting the API call for the info?

Reply:
@Java-Jason10611